substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2538,7 +2556,12 @@ pub mod pallet {
 				Error::<T>::MinimumBondNotMet
 			);
 
-			T::StakeAdapter::nominate(Pool::from(bonded_pool.bonded_account()), validators)
+			let nominate_res = T::StakeAdapter::nominate(Pool::from(bonded_pool.bonded_account()), validators);
+			if let Ok(_) = nominate_res {
+				Self::deposit_event(Event::<T>::PoolNominationMade { pool_id, caller: who })
+			}
+
+			return nominate_res
```

Comment:
Another way to write this would be using combinators. 
(unformatted)
```suggestion
    T::StakeAdapter::nominate(Pool::from(bonded_pool.bonded_account()), validators)
				.map(|_| {
					Self::deposit_event(Event::<T>::PoolNominationMade { pool_id, caller: who })
				})
```